packages/raystack/v1/typings.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module "*.module.css" {
```

Comment:
Kept this in v1 as well as v1 is nested in dist. Also, the outer typings file contains extra exports which are not needed for the v1. It will be easier to identify which ones to keep when we will be removing the older components.